### DISCUSSION TITLE
Why Borg Backup Container Timezone is different from main global timezone ?

### DISCUSSION THREAD

**Discussion by tdeseez** [nextcloud/all-in-one#2806] [Questions] [answered]
Hi

I specified general timezone to be set on Europe/Paris during AIO installation, and therefore Borg backup timezone is still set to UTC. 
It's deeply confusing.

Seems to affect any AIO version i tried so far.

Is there a reason why or is this kind of a bug ? 
Is there anyway to set borg container timezone ? 

Thanks for your answers.

PS : AIO is awesome

**Comment by szaimen** [comment, marked as answer]
> Is there a reason why or is this kind of a bug ?
> Is there anyway to set borg container timezone ?

Adjusting the timezone for the backup container is prevented by purpose because the timestamp of each backup is the main identifier of each backup. So if we would allow to adjust the timezone for borg, you could run into a situation where the last backup has a timestamp that is older then the one from the backup that you created before it which would lead to you restoring the wrong backup if you simply want to restore the last one.

**Reply by Chouffy** [reply]
Hey @szaimen 

I would like to challenge this on three grounds:
- Having Borg in UTC will brings some issues during the Summer/Winter time change: all my other automation uses the local TZ, and I would need to change Nextcloud AIO so it doesn't start one hour earlier.
- The timezone is already adjusted in the mastercontainer, so it's unlikely that the user will change it later on. I agree that's not impossible, but the occurrence would be very rare (= a relocation?).
- Calculating times in UTC isn't very convenient, and the default timing at 04:00 UTC is good for Europe and Americas … but not for Asia that will experience downtime during daytime. Of course the admin could change the default time, but you could miss the info that's UTC and not local TZ

Would it be possible to have a flag for it?

BTW thanks a lot for your hard work on this, it's really awesome ! 😊

**Comment by tdeseez** [comment]
Thank you for answering. Not sure to understand everything though, but now i know i'll have to scratch ma head everytime i have to deal with backup times ! 

So if want to have my backup launched at 06:00 am french time, i'll have to set it to 04:00 UTC, right ?

keep up good work.
regards/

**Reply by szaimen** [reply]
> So if want to have my backup launched at 06:00 am french time, i'll have to set it to 04:00 UTC, right ?

Yes
